pulp_fedora31/Containerfile

```diff
@@ -7,6 +7,7 @@ ARG PULP_ANSIBLE_VERSION=""
 ARG PULP_DEB_VERSION=""
 ARG PULP_RPM_VERSION=""
 ARG PULP_MAVEN_VERSION=""
+ARG PULP_PYTHON_VERSION="==3.0.0b12"
```

Comment:
Why are you specifically hardcoding the pulp_python version?

Reply:
I guess, because it has not been officially released yet.
And i also thought, we only included released Plugins here, because the upgrade path would be horrifying.